core/controllers/collection_editor.py

```diff
@@ -61,7 +61,7 @@ def get(self, collection_id):
 
         self.values.update({
             'collection_id': collection.id,
-            'nav_mode': feconf.NAV_MODE_CREATE,
+            'nav_mode': 'create',
```

Comment:
Do you need this variable anymore? This is just passed to the Jinja and Jinja will include int the HTML files. And since you removed NAV_MODE from base.html it!s not needed anymore. Ditto for the other files below.

Reply:
@vojtechjelinek Oh. Removing these then.
Thanks.

Reply:
@vojtechjelinek Does this look fine now?

Reply:
Yes